presto-product-tests/bin/run_on_docker.sh

```diff
@@ -70,25 +70,46 @@ function stop_application_runner_containers() {
   done
 }
 
+function stop_all_containers() {
+  local ENVIRONMENT
+  for ENVIRONMENT in $(getAvailableEnvironments)
+  do
+     stop_docker_compose_containers
+  done
+}
+
 function stop_docker_compose_containers() {
-  RUNNING_CONTAINERS=$(docker-compose -f "$1" ps -q)
+  RUNNING_CONTAINERS=$(environment_docker_compose ps -q)
 
   if [[ ! -z ${RUNNING_CONTAINERS} ]]; then
     # stop application runner containers started with "run"
-    stop_application_runner_containers "$1"
+    stop_application_runner_containers
 
     # stop containers started with "up"
-    docker-compose -f "$1" down
+    environment_docker_compose down
   fi
 
   echo "Docker compose containers stopped: [$1]"
 }
 
+function environment_docker_compose() {
+  local ENVIRONMENT_DOCKER_COMPOSE_LOCATION="${DOCKER_CONF_LOCATION}/${ENVIRONMENT}/docker-compose.yml"
+  local COMPOSE_FILES=( -f "${DOCKER_CONF_LOCATION}/common/standard.yml" )
+  if [[ $ENVIRONMENT == *"kerberos"* ]]; then
+    COMPOSE_FILES+=( -f "${DOCKER_CONF_LOCATION}/common/kerberos.yml" )
+  fi
+  COMPOSE_FILES+=( \
+    -f "${DOCKER_CONF_LOCATION}/common/jdbc_db.yml" \
+    -f "${ENVIRONMENT_DOCKER_COMPOSE_LOCATION}" \
+  )
+  docker-compose "${COMPOSE_FILES[@]}" "$@"
+}
```

Comment:
@kokosing and @fiedukow actually convinced me it's better outside of this file for usability reasons. Often we need to execute `docker-compose whatever` in a given directory / by passing a single `-f` argument and having this dependencies expressed in `run_on_docker` makes doing anything else than running the tests really hard.

I'm hesitating between two approaches: having a `compose.sh` oneliner in each of env dirs or writing a `docker-composer` script in `bin` that would read `#requires ../common/standard.yml` directives from the files recursively. I think I'll start with the former.